dist/build.gradle

```diff
@@ -79,15 +79,21 @@ task distDir(group: 'Build',
              description: "Builds a distribution directory (${project.ext.relativeDistDir})",
              dependsOn: [tasks.copyLicenses, tasks.copyLib, tasks.copyClientBinaries])
 
+task distDirWithoutClientBinaries(group: 'Build',
+        description: "Builds a distribution directory without client binaries (${project.ext.relativeDistDir})",
+        dependsOn: [tasks.copyLicenses, tasks.copyLib])
+
 // Create the tasks that copy each directory under src/ into dist/
 ['bin', 'conf'].each { dirName ->
     def taskName = "copy${CaseFormat.LOWER_HYPHEN.to(CaseFormat.UPPER_CAMEL, dirName)}"
-    tasks.distDir.dependsOn(tasks.create(taskName, Copy) {
+    def copyTasks = tasks.create(taskName, Copy) {
```

Comment:
- copyTasks -> copyTask
- With my earlier comment, you wouldn't need to define a local variable here.